app/assets/v2/js/pages/increase_bounty.js

```diff
@@ -211,27 +214,29 @@ $(document).ready(function() {
       const fee = (Number($('#summary-bounty-amount').html()) * FEE_PERCENTAGE).toFixed(4);
 
       indicateMetamaskPopup();
-      if (isETH) {
-        web3.eth.sendTransaction({
-          to: to_address,
-          from: web3.eth.coinbase,
-          value: web3.toWei(fee, 'ether'),
-          gasPrice: gas_price
-        }, function(error, txnId) {
-          indicateMetamaskPopup(true);
-          if (error) {
-            _alert({ message: gettext('Unable to pay bounty fee. Please try again.') }, 'error');
-          } else {
-            // TODO: Save txnId + feeamount to bounty;
-            do_as_funder();
-          }
-        });
+      if (FEE_PERCENTAGE == 0) {
+        do_as_funder();
       } else {
-        const amountInWei = fee * 1.0 * Math.pow(10, token.decimals);
-        const token_contract = web3.eth.contract(token_abi).at(tokenAddress);
+        if (isETH) {
+          web3.eth.sendTransaction({
+            to: to_address,
+            from: web3.eth.coinbase,
+            value: web3.toWei(fee, 'ether'),
+            gasPrice: gas_price
+          }, function(error, txnId) {
+            indicateMetamaskPopup(true);
+            if (error) {
+              _alert({ message: gettext('Unable to pay bounty fee. Please try again.') }, 'error');
+            } else {
+              // TODO: Save txnId + feeamount to bounty;
```

Comment:
does this TODO still need to be added?

Reply:
This would be a separate task! 
This would involve updating the bounty model / creating a new model to keep track of the fees.

I had brought this up when we originally built out fee%.
As of now we have no means to store and track it. 
Here I've simple reorganized the code

need to evaluate if it would sense to keep fee in it's model and map bounties and other produxts to it. cc @owocki